### ISSUE TITLE
Not all classes are being pulled in from Blade files, some only after re-saving the files.

### ISSUE THREAD

**Issue by foof** [tailwindlabs/tailwindcss-jit#110]
### What version of @tailwindcss/jit are you using?

0.1.3

### What version of Node.js are you using?

v14.16.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### Reproduction repository

https://github.com/foof/tailwind-jit-test

The linked repo is a fresh Laravel install but where all the build configs are located in `resources/myfolder/` instead of the root folder.

Reproduction steps:

1. Clone repo
2. `cd resources/myfolder`
3. `npm install`
4. `npm run watch`

The purge option is configured to look at blade files in the `resources/views/myfolder` directory.

Classes from `resources/views/myfolder/welcome.blade.php` are not being put into the generated CSS file. If I simply re-save the blade file while watch is running, then some of the classes gets added, but not all. For example `sm:p-6` is never included in the generated css.

**Comment by JawadR1** [comment]
You don't have package.json or package-lock.json in your repo. `npm install` won't work.

**Comment by foof** [comment]
> You don't have package.json or package-lock.json in your repo. `npm install` won't work.

Perhaps it was unclear, but those files are also located in the the subfolder `resources/myfolder`. In our project we have multiple frontends which means we have multiple package.json etc that are located in other subfolders like `resources/myfolder`, `resources/myfolder2`, `resources/myfolder3` etc.

**Comment by foof** [comment]
That explains some of the classes :) However `w-96` is not included when I first start `npm run watch` after making your suggested changes. But if I simply go into the blade file and hit Cmd+S without changing anything it then recompiles and `w-96` is included

**Comment by RobinMalfait** [comment]
Very odd, I am going to re-open this issue. It seems that when you drop the `"./**/*.{js,vue}"` line it does create the `w-96` consistently on the first run. I do have to mention that I see multiple "runs" in the webpack output.
When you include that line, initially the `w-96` does not get created. 

The good part is that I can reproduce it consistently.
Once I find the solution / actual issue I'll update this issue!

Thanks for opening the issue!

**Comment by foof** [comment]
I also get multiple runs which I forgot to mention and I wasn't sure if it was intended or not. Normally we also build JS but I removed all non-CSS stuff from the repo for simplicity's sake.

If it helps, an interesting thing I found is that if I copy the entire `views` directory into `resources/myfolder` and change the purge config to `'./views/**/*.php'`, then it works as expected.

**Comment by bakerkretzmar** [comment]
I'll open another issue but I just want to add that I'm running into the same thing, specifically multiple compilations and classes from Blade files missing after the first save.

I'm using a pretty standard Laravel Mix setup, and with JIT _every_ Mix compilation runs twice now. It may be because both Mix and JIT are watching a bunch of the same files, but I need them to since I have Tailwind classes in my Vue components.

**Comment by bakerkretzmar** [comment]
Okay, inexplicably it's now working fine. I set `DEBUG=true` to see the JIT output, everything seemed fine, and I removed it again and everything now works as expected, even the double builds are gone. Gonna leave my comment in case it helps someone else, but sorry to muddy the waters!

**Comment by Syakhisk** [comment]
I was having this problem after I added the `TAILWIND_MODE=watch` variable, not sure why but after I removed it the classes get generated normally.
